android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativeModule.kt

```diff
@@ -67,7 +72,7 @@ class NativeCustomerIOModule(
                 ?.let { flushInterval(it) }
                 (packageConfig[Keys.Config.TRACK_APP_LIFECYCLE_EVENTS] as? Boolean)
                 ?.let { trackApplicationLifecycleEvents(it) }
-            // TODO: Implement pushClickBehaviorAndroid when initializing messagingModule
+                (addCustomerIOModule(module = configureModuleMessagingPushFCM(packageConfig)))
```

Comment:
```suggestion
                addCustomerIOModule(module = configureModuleMessagingPushFCM(packageConfig))
```

why do we have an extra set of brackets?

Reply:
Oh yes, it shouldn't be there.